src/services/db/RepoService.ts

```diff
@@ -592,14 +618,14 @@ export default class RepoService extends GitHubService {
     await super.updateRepoState(sessionData, { commitSha })
   }
 
-  async checkHasAccess(sessionData: any): Promise<any> {
+  async checkHasAccess(sessionData: UserWithSiteSessionData): Promise<void> {
     return super.checkHasAccess(sessionData)
   }
 
   async changeRepoPrivacy(
-    sessionData: any,
-    shouldMakePrivate: any
-  ): Promise<any> {
+    sessionData: UserWithSiteSessionData,
+    shouldMakePrivate: boolean
+  ): Promise<ResultAsync<null, NotFoundError | GitHubApiError>> {
```

Comment:
nit: same here regarding the wrapping of `ResultAsync`

Reply:
Same as the[ comment here](https://github.com/isomerpages/isomercms-backend/pull/1066#discussion_r1432225854)

Reply:
Refactored in 1cb950bcd9aa3b41314912c8e30bb51cfc6e9c52 along with [the comment here](https://github.com/isomerpages/isomercms-backend/pull/1066#discussion_r1432250037)